### PR TITLE
Add the option to skip installing packages

### PR DESCRIPTION
**Pull request by ehankland**
Add the option to skip installing packages (for static vms only)

### CODE REVIEW

**Comment by ehankland** [comment]
test this please.

File: perfkitbenchmarker/package_managers.py [hunk 1]
```diff
@@ -161,6 +161,8 @@ def InstallPackageGroup(self, package_group):
 
   def Install(self, package_name):
     """Installs a PerfKit package on the VM."""
+    if self.is_static and not self.install_packages:
```
Comment (voellm) [inline perfkitbenchmarker/package_managers.py]: Is these a reason to restrict this to only static VMs?  I can see this being useful for even custom images.

File: perfkitbenchmarker/package_managers.py [hunk 2]
```diff
@@ -237,6 +239,8 @@ def InstallPackages(self, packages):
...
```
Comment (voellm) [inline perfkitbenchmarker/package_managers.py]: Is these a reason to restrict this to only static VMs? I can see this being useful for even custom images.

**Comment by voellm** [comment]
Added some comments in the code.  Just ping me to look again once you decide how to go with the feature.

**Comment by ehankland** [comment]
Added a flag override - please take another look

**Comment by voellm** [comment]
LGTM

**Pull request merged**